main.js

```diff
@@ -520,6 +520,17 @@ function windowCreateFinish(json) {
             url: url
         })
     })
+    if (typeof json.windowOptions.appDetails !== "undefined"){
+        elements[json.targetID].setThumbarButtons([]);
+        elements[json.targetID].setAppDetails({
```

Comment:
You don't have to build a new object here, you can feed `json.windowOptions.appDetails` directly to `elements[json.targetID].setAppDetails`